### ISSUE TITLE
S3 backend has multiple states

### ISSUE THREAD

**Issue by ep4sh** [gruntwork-io/terragrunt-infrastructure-live-example#32]
Dear team,
accordingly https://github.com/gruntwork-io/terragrunt-infrastructure-live-example/blob/master/terragrunt.hcl#L43
Due to each module located in separated dir, would each module have its own .tfstate (one per module)? Is it correct?

**Comment by yorinasub17** [comment]
Yes that is correct. This is intentional, and one of the strengths of terragrunt (making it easy to manage modules with different state files).

**Comment by ep4sh** [comment]
Sounds good, thanks for the clear answer.
